lib/Graph/CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ add_custom_target(AutoGenNode
                      "${NODES_DEF}")
 add_dependencies(AutoGen AutoGenNode)
 
+
```

Comment:
Was this blank line intentionally added?

Reply:
removed now :P